software/control/gui_hcs.py

```diff
@@ -990,12 +995,13 @@ def makeConnections(self):
 
         if SUPPORT_LASER_AUTOFOCUS:
 
-            def connect_objective_changed_laser_af():
-                self.laserAutofocusController.on_objective_changed()
+            def connect_settings_changed_laser_af():
```

Comment:
change `connect_settings_changed_laser_af ` to `slot_settings_changed_laser_af `?